packages/ui-kit/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Mostly from the intial automatic migration plus some the emotion thing I added to get emotion run. 

Maybe we have a template or standard in the company from like for eslint?

Reply:
Only the commented code got removed, right?